azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java

```diff
@@ -29,6 +34,8 @@
  * Azure Compute sample for managing virtual machine scale sets -
  *  - Create a virtual machine scale set behind an Internet facing load balancer
  *  - Install Apache Web servers in virtual machines in the virtual machine scale set
+ *  - List the network interfaces associated with the virtual machine scale set
+ *  - List scale set virtual machine instances and SSH collection string
```

Comment:
If this change is for current sample you need to update the individual sample repo as well.